### PR TITLE
Separated geometric and dynamic computations.

### PR DESCRIPTION
**Pull request by NahuelVilla**
I have separated the computation of the centroidal dynamics in a new class.

* The functions to compute the dynamics are still in `biped_ig`, but they should be removed.
* Eventually the class to compute the dynamics can be in another package (this is to be decided later.)
* This branch will be finished with the force distributor, but it can be already merged for the separated computations.

### CODE REVIEW

**Comment by NahuelVilla** [comment]
This is working and ready to merge.

File: python/biped_ig.cpp [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+#include <boost/python/suite/indexing/vector_indexing_suite.hpp>
+#include <eigenpy/eigenpy.hpp>
+
+#include "aig/biped_ig.hpp"
+#include "aig/python.hpp"
+
+namespace aig {
+namespace python {
+namespace bp = boost::python;
+
```
Comment (MaximilienNaveau) [inline python/biped_ig.cpp:16]: add methods for solve overload here:

```
bp::tuple solve1(BipedIG& self, args)
{
   Eigen::VectorXd pos, vel, acc;
   self.solve(args, pos, vel, acc)
   return bp::tuple(pos, vel, acc)
}
```

**Review by MaximilienNaveau — COMMENTED** [review]
update the solve bindings

**Comment by pFernbach** [comment]
It's important to note that this PR add a dependency to proxsuite (https://github.com/Simple-Robotics/proxsuite). This package doesn's seems to provide binaries on 18.04. 
Also, this force aig to compile in C++ 17.

**Comment by NahuelVilla** [comment]
Yes, it is not needed for the geometric part. 
We have planned with Max to separate the dynamic part in another package because they are two independent computations. These dependencies are another reason to do the separation

**Pull request closed without merging**